src/main/row.ts

```diff
@@ -3,7 +3,7 @@ import { InvalidHashError } from "./error/invalidHashError";
 import { RowBasecampMappingMissingError } from "./error/rowBasecampMappingMissingError";
 import { RowMissingIdError } from "./error/rowMissingIdError";
 import { RowNotSavedError } from "./error/rowNotSavedError";
-import { FILTER_NAMES, filterMembers, getFilterNames } from "./filter";
```

Comment:
This was throwing a compiler error because the function didn't exist causing the tests to freak out